R/bind_pdf.r

```diff
@@ -0,0 +1,32 @@
+#' Binds together the PDFs from an array job - is called by 'pdf_array_job()'. 
+#'
+bind_pdfs = function(temp_out_dir, final_out_dir, write_file_name){
+  require(grid)
+  require(gridExtra)
+  require(png)
+  
+  files = list.files(temp_out_dir, full.names=T, include.dirs=F)
+  
+  pdf(paste0(final_out_dir, "/", write_file_name))
+  for (file in files){
+    grob = rasterGrob(readPNG(file, native = FALSE), interpolate = FALSE)
+    grid.arrange(grob)
+  } 
+  dev.off()
+} 
+
+parser <- argparse::ArgumentParser()
+parser$add_argument("--temp-out-dir", required=TRUE, help="Temporary directory where files are saved")
+parser$add_argument("--final-out-dir", required=TRUE, help="Final directory to save files in")
+parser$add_argument("--write-file-name", required=TRUE, help="Final PDF name to write")
+args <- parser$parse_args()
+
+temp_out_dir = args$temp_out_dir
+final_out_dir = args$final_out_dir
+write_file_name = args$write_file_name
+
+print(paste0("Temporary output directory: ", temp_out_dir))
+print(paste0("Final output directory: ", final_out_dir))
+print(paste0("File name to write: ", write_file_name))
+
+bind_pdfs(temp_out_dir, final_out_dir, write_file_name)
```

Comment:
Lines 18-32 can't stay as-is because they'd try and run when the package was loaded.

This looks like a template for teaching users how to write their own script. What if you added it to the `README.md` entry for `bind_pdfs`?

Reply:
Yes it is - that could work. Or I could also post on a hub page so I don't clutter up the README?

Reply:
That would be fine, as would be creating an additional `.md` document and linking to it e.g., (from `README.md`) adding

```
see also [the full documentation](awesome_bind_pdfs_doc.md)
```